### DISCUSSION TITLE
Release 1.9.8

### DISCUSSION THREAD

**Discussion by mattwthompson** [mdtraj/mdtraj#1793] [General]
I've tagged version 1.9.8 and uploaded the [sdist to PyPI](https://pypi.org/project/mdtraj/1.9.8/#files). Conda packages are [online soon](https://prefix.dev/channels/conda-forge/packages/mdtraj).

This release should support Python 3.11 and the `osx-arm64` architecture.

Please use this discussion to report any deployment issues with this version specifically.

**Comment by sukritsingh** [comment]
I guess just for tracking purposes it looks like there appears to have been an issue with Cython (sorry was out last week)? The bandaid fix constraint is great (thanks for that!) but for a longer-term cython support do we want to push that within this release or push it to a future release?

**Comment by mattwthompson** [comment]
Yup - Cython dropped version 3.0.0 on Monday (a notable jump up from the previous 0.29.36!) and I'm planning to get 1.9.9 tagged within the next day or so.

Tracked in #1800 for more context, the short version being:

## **If you are getting errors with `pip install mdtraj`, add `Cython~=0.29` and version 1.9.8 (possibly olders as well) should work**

**Reply by sukritsingh** [reply]
> Yup - Cython dropped version 3.0.0 on Monday (a notable jump up from the previous 0.29.36!) and I'm planning to get 1.9.9 tagged within the next day or so.

Sounds good! Thanks for driving the fix – Just to clarify: would your plan here be that 1.9.9 supports cython 3.0.0 or that 1.9.9 just constraints to cython 0.29?

**Reply by sukritsingh** [reply]
Ah I see ``Pin `Cython~=0.29` in `pyproject.toml`` from #1801, never mind.

In the long term supporting Cython 3.0.0 makes sense right? I don't know what the effort cost there would be. I haven't actually seen what the update breaks yet (and admittedly, I need to become more familiar with cython) but let me know if I can help with that updating effort.

**Comment by mattwthompson** [comment]
The idea is 1.9.9 is the same source code but includes the Cython downpin in `pyproject.toml` so PyPI installs work. (Conda installs have been patched.) We should absolutely follow that up with a new release (1.9.10 or 1.1.0) that works with Cython 3 once we get that patch through. I haven't looked at what's needed; hopefully it's not too much. I have not worked much with Cython, but it's a big project so, optimistically, the docs will provide us with enough guidance.

This isn't necessarily the most elegant course of action but I think it's clearly the most practical one. The ideal situation is everything is fixed quickly and we can push those updates out into new releases rapidly, but that's not guaranteed to happen in the timespan of a couple of days. Getting `pip install mdtraj` working out of the box sooner than later is my main objective here.

**Comment by sukritsingh** [comment]
> This isn't necessarily the most elegant course of action but I think it's clearly the most practical one. [...] Getting pip install mdtraj working out of the box sooner than later is my main objective here.

100% agree that this is the best course of action - getting a working version with 1.9.9 followed by a Cython 3 compatible update makes the most sense to me! It'll also give us the flexibility of time to figure out the fix
